python/pyspark/sql/readwriter.py

```diff
@@ -300,18 +300,20 @@ def table(self, tableName):
         return self._df(self._jreader.table(tableName))
 
     @since(1.4)
-    def parquet(self, *paths):
```

Comment:
Side question for you @HyukjinKwon: The `*paths` parameter bothers me a bit. None of the other load methods use this pattern, and the streaming version of `parquet()` doesn't use it either. How would you feel about a separate PR changing this to `paths`? I suppose the 3.0 release would be our chance to do it, since it changes the API.

Reply:
I think this `*paths` allows a consistent way with Scala and Java side (`def parquet(paths: String*)`). So I think technically it's more correct to support `*paths`.

In case of streaming, it's also matched to Scala / Java side `def parquet(path: String)`.

Maybe we should introduce keyword-only argument after completely dropping Python 2 in Spark 3.1. ... I am not sure about this yet.